src/Bundle/CoverallsBundle/Collector/CiEnvVarsCollector.php

```diff
@@ -110,6 +111,34 @@ protected function fillTravisCi()
         return $this;
     }
 
+    /**
+     * Fill Github Actions environment variables.
+     *
+     * @return $this
+     */
+    protected function fillGithubActions()
```

Comment:
Would be nice to keep the method order  to comply with order in `collect()`.

Reply:
Do you mean that call `fillGithubActions` should last?

Reply:
yes